docs/UserGuide.md

```diff
@@ -327,14 +387,18 @@ An example of how the CSV should be formatted is shown below:
 
 ![CsvFormatExample](images/CsvFormatExample.png)
 
+The command to import this file would be `import -f student_data.csv -g 2 -a 2 -t 2`. (There are 2 group columns, 2 assessments, and 2 tag columns.)
+
 Examples:
-* `import -f /home/prof/CS1101S/student_data.csv -g 2 -a 10 -t 1` imports from the absolute path and database has 2 group columns, 10 assessments, and 1 tag column.
-* `import -f student_data.csv -g 5 -a 20` imports from the relative path and database has 5 group columns, 20 assessments.
+* `import -f /home/prof/CS1101S/student_data.csv -g 2 -a 10 -t 1` 
+  * imports from the absolute path and database has 2 group columns, 10 assessments, and 1 tag column.
+* `import -f student_data.csv -g 5 -a 20` 
+  * imports from the relative path, and database has 5 group columns and 20 assessments.
```

Comment:
How about this?

```suggestion
  * imports from the absolute path, and the database has 2 group columns, 10 assessment columns, and 1 tag column.
* `import -f student_data.csv -g 5 -a 20` 
  * imports from the relative path, and the database has 5 group columns and 20 assessment columns.
```

Reply:
actually importing assessments works differently from group. i'm trying to reinforce that the group columns =! number of groups. using `assessment columns` works too, if u think this format is too confusing

Reply:
Oh I see. I'm just afraid the ones who will be reviewing our UGDG may spot this. In that case, as you say they work differently so we can explain it later, but hmm if `assessment columns` is not wrong then I think we should make them consistent (since the reviewers won't know that they work differently)

Reply:
ok sure